template/header.html

```diff
@@ -0,0 +1,32 @@
+<nav class="navbar navbar-expand-lg navbar-light  bg-light " id="mainNav" role="navigation">
+    <div class="container">   
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+      </button>
+      <a class="navbar-brand" href="#"><img class="logo" src="images/logo.png"></a>
+      <div class="collapse navbar-collapse" id="navbarTogglerDemo03">
+        <ul class="navbar-nav ml-auto mt-2 mt-lg-0">
+          <li class="nav-item ">
+            <a class="nav-link" href="index.html">Home <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item active">
+                  <a class="nav-link" href="#">About </a>
```

Comment:
@Orung since we have the about us page up, can you update the link to the about us page here.

Reply:
Am sure she had done this, This probably is a result of previous commits.